### PR TITLE
Add Series.skew/2

### PR DESCRIPTION
**Pull request by guarilha**
https://pola-rs.github.io/polars/py-polars/html/reference/series/api/polars.Series.skew.html

### CODE REVIEW

File: lib/explorer/series.ex [hunk 1]
```diff
@@ -1864,6 +1864,53 @@ defmodule Explorer.Series do
   def quantile(%Series{dtype: dtype}, _),
     do: dtype_error("quantile/2", dtype, [:integer, :float, :date, :time, :datetime])
 
+  @doc """
+  Compute the sample skewness of a data set.
+
+  For normally distributed data, the skewness should be about zero. For unimodal continuous distributions, a skewness value greater than zero means that there is more weight in the right tail of the distribution. The function skewtest can be used to determine if the skewness value is close enough to zero, statistically speaking.
```
Comment (josevalim) [inline lib/explorer/series.ex:1870]:
```suggestion
  For normally distributed data, the skewness should be about zero.
  For unimodal continuous distributions, a skewness value greater
  than zero means that there is more weight in the right tail of the
  distribution. The function skewtest can be used to determine if
  the skewness value is close enough to zero, statistically speaking.
```

File: lib/explorer/series.ex [hunk 2]
```diff
@@ -1864,6 +1864,53 @@ defmodule Explorer.Series do
...
+
+  See scipy.stats for more information.
```
Comment (josevalim) [inline lib/explorer/series.ex:1872]: We should not point to scipy :D

File: lib/explorer/series.ex [hunk 3]
```diff
@@ -1864,6 +1864,57 @@ defmodule Explorer.Series do
   def quantile(%Series{dtype: dtype}, _),
     do: dtype_error("quantile/2", dtype, [:integer, :float, :date, :time, :datetime])
 
+  @doc """
+  Compute the sample skewness of a data set.
```
Comment (josevalim) [inline lib/explorer/series.ex:1868]:
```suggestion
  Compute the sample skewness of a series.
```

File: lib/explorer/series.ex [hunk 4]
```diff
@@ -1864,6 +1864,57 @@ defmodule Explorer.Series do
...
+
+  For normally distributed data, the skewness should be about zero.
+  For unimodal continuous distributions, a skewness value greater
+  than zero means that there is more weight in the right tail of the
+  distribution. The function skewtest can be used to determine if
```
Comment (josevalim) [inline lib/explorer/series.ex:1873]: What is the `skewtest` function?

File: lib/explorer/series.ex (hunk 2 above)
Comment (guarilha) [reply lib/explorer/series.ex:1876]: haha my bad, copied the description from polars and forgot to edit it. I'll change it and the ref to skewtest.

File: lib/explorer/series.ex (hunk 4 above)
Comment (guarilha) [reply lib/explorer/series.ex]: Removed the reference to `skewtest` as it seems to just be available on [scipy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.skewtest.html#scipy.stats.skewtest).

**Comment by josevalim** [comment]
Looking good now! Can you please look at the CI now? It seems we are missing `skew` in the lazy backend (and we need to add a test for it like in the previous PRs).

**Review by philss — APPROVED** [review]

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**